protocol/0044-LIME-lp_mechanics.md

```diff
@@ -19,87 +59,81 @@ Any Vega participant can apply to become a liquidity provider (LP) on a market b
   - Transfers to/from this account also occur when it is used for settlement or margin shortfall, when penalties are applied, and if the account is under-collateralised because of these uses and is subsequently topped up to the commitment amount during collateral search (see below)
 - Collateral withdrawn from this account may only be transferred to either:
   - The insurance pool of the market (in event of penalties/slashing)
   - The liquidity provider's margin account or the network's settlement account/other participant's margin accounts (during a margin search and mark to market settlement) in the event that they have zero balance in their general account.
   - The liquidity provider's general account (in event of liquidity provider reducing their commitment)
 
-### liquidity provider proposes to amend commitment amount
+### Liquidity provider proposes to amend commitment amount
 
 The commitment transaction is also used to amend any aspect of their liquidity provision obligations.
 A participant may apply to amend their commitment amount by submitting a transaction for the market with a revised commitment amount.
 
 `proposed-commitment-variation = new-proposed-commitment-amount - old-commitment-amount`
 
-#### INCREASING COMMITMENT
+The amendment is actioned at the beginning of the next epoch (after the rewards / penalties for present LPs - including the party that's amending - have been evaluated). It's the amending LP's responsibility to have sufficient balance in their general account at the epoch boundary; if not the LP amendment is not actioned.
+
+#### Increasing commitment
 
 _Case:_ `proposed-commitment-variation >= 0`
-A liquidity provider can always increase their commitment amount as long as they have sufficient collateral in the settlement asset of the market to meet the new commitment amount and cover the margins required.
+A liquidity provider can always increase their commitment amount as long as they have sufficient collateral in the settlement asset of the market to meet the new commitment amount.
 
-If they do not have sufficient collateral the transaction is rejected in entirety. This means that any data from the fees or orders are not applied. This means that the  `old-commitment-amount` is retained.
+If they do not have sufficient collateral the transaction is rejected in entirety. This means that any change in fee bid is not applied and that the `old-commitment-amount` is retained.
 
-#### DECREASING COMMITMENT
+#### Decreasing commitment
 
 _Case:_ `proposed-commitment-variation < 0`
-We to calculate whether the liquidity provider may lower their commitment amount and if so, by how much. To do this we first evaluate the maximum amount that the market can reduce by given the current liquidity demand in the market.
+We to calculate how much the LP can reduce commitment without incurring a penalty. 
```

Comment:
Writing the ACs for this a further thought has come up. If we now only process these reductions in commitment at the end of an epoch there's a question of sequencing and thus penalties which might not be immediately obvious when first placing your reduction transaction (e.g. if someone else reduces their commitment such that the stake is much closer to target than it was when you first placed your reduction). 

The 'easiest' solution would just be to say we action these in the order they are placed, and LPs should just take this into account when they place their reductions, but we should probably then at least be explicit somewhere that reductions are executed in order of receipt.

Reply:
@TomMcL that's a good point, lets add a line saying that reductions are processed in the sequence they were placed; for each LP it's the most recent that counts.